### PR TITLE
docs: Add Breaking Change Book

### PR DESCRIPTION
**Pull request by calldelegation**
https://github.com/FuelLabs/breaking-change-log

### CODE REVIEW

**Comment by calldelegation** [comment]
Remove old breaking change log guide

File: scripts/generate-links/getDocs.mjs [hunk 1]
```diff
@@ -64,6 +64,24 @@ export async function getDocs(key, order) {
         '!**/SUMMARY.md',
       ];
       break;
+    case 'breaking-change-log':
+      paths = [
+        // BREAKING CHANGE LOG DOCS
+        './breaking-change-logdocs/src/*.md',
```
Comment (calldelegation) [inline scripts/generate-links/getDocs.mjs:70]:
```suggestion
        './breaking-change-log/docs/src/*.md',
```

File: contentlayer.config.ts [hunk 2]
```diff
@@ -2,48 +2,48 @@ import {
   defineDocumentType,
   defineNestedType,
   makeSource,
-} from 'contentlayer/source-files';
+} from "contentlayer/source-files";
```
Comment (matt-user) [inline contentlayer.config.ts:5]: Did we change the formatter settings?

**Review by matt-user — COMMENTED** [review]
Still have to look at the deployment but looks good

File: contentlayer.config.ts (hunk 2 above)
Comment (calldelegation) [reply contentlayer.config.ts:5]: Yea very messy. Ill fix this on my own machine and change it back

**Comment by matt-user** [comment]
The link on the summary page for breaking changes is 404.  Also the link check in the ci seems to be flagging something w beta-4

**Comment by calldelegation** [comment]
https://github.com/FuelLabs/sway/blob/c186d93155d0ef9f4432773be08c7da6d0fdf6de/docs/book/src/forc/plugins/forc_client/index.md?plain=1

Found the issue I will fix this in the Sway repo

**Review by Nazeeh21 — COMMENTED** [review]
Overall it LGTM! Just the inverted commas need to be fixed just like Matt mentioned, other than that it's perfect!

**Review by Nazeeh21 — APPROVED** [review]
Changes LGTM! This PR is ready to be merged

**Pull request merged**